src/main/resources/application.properties

```diff
@@ -9,3 +9,6 @@ spring.jpa.hibernate.ddl-auto=validate
 spring.jpa.show-sql=true
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQLDialect
 spring.jpa.open-in-view=false
+
+jwt.expiration= 300000
+jwt.secret=secretmessagemfdkfgdpwe32fosj08sjsnflwjtphsowh3ip28rnbsfk285twsdfjnweo
```

Comment:
Please don`t push sensitive information

Reply:
let's provide this value from env properties